download.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
+const publicKey = process.argv[2]
+const privateKey = process.argv[3]
+const _ = require('underscore')
+
+const onesky = require('onesky')(publicKey, privateKey)
+const sanitizeHtml = require('sanitize-html')
+const sanitizeOpts = {
+  allowedTags: ['b', 'strong', 'a', 'code'],
+  allowedAttributes: {
+    a: ['href', 'class']
+  },
+  textFilter(text) {
+    text.replace(/\{\{/, '&#123;&#123;').replace(/\}\}/, '&#125;&#125;')
+  }
+}
+
+onesky.string.output({ platformId: '25049' }, function(err, r) {
+  if (err) {
```

Comment:
This `if (err) { }` block was caught by the linter, I assume it was an error, not a workaround for a bug in onesky.

Reply:
There was not error handing in the coffee-script version, but it's a good idea to add an error handler.

https://github.com/overleaf/translations/blob/f5de21d70a1fe60835ee7ccf7f27c8c03dbfd7f3/download.coffee#L15-L16